app/main.py

```
@@ -0,0 +1,1171 @@
+            case_id=case.id,
+            details=", ".join(change_descriptions),
+        )
+
+    return CaseRead.from_orm(case)
+
+
+@app.put("/cases/{case_id}/outcome", response_model=CaseOutcomeRead)
+def upsert_case_outcome(
+    case_id: int,
+    payload: CaseOutcomeCreate,
+    session: Session = Depends(get_session),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)